manifests/image-references

```diff
@@ -66,3 +66,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-thanos:latest
+  - name: console-plugin
```

Comment:
sorry I think this needs to be `monitoring-plugin` since that is what the image is being promoted as.

Reply:
I see! thank you!